crypto.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import re
+
+from decorators      import is_command
+from crypto_data     import crypto_data
+
+def get_key(value):
+    for key, value in crypto_data.items():
+        if value == value:
+            return key
+
+    return "BTC"
+
+@is_command("Check all the crypto prices.")
+def crypto(message_dict):
+    user = message_dict["display-name"].lower()
+    message = [message_dict["message"]]
+    message_dict["suppress_log"] = True
+    
+    # Attempt to split the message and get each item into an array
+    try:
+        message = re.split(', |; |\s', message)
+    # Fail gracefully if it's a single or empty message
+    except Exception as ex:
+        log(f"Exception in crypto.crypto: {str(ex)}")
+
+    # Check each item requested and make sure it's the short-code and not the full name
+    # Convert to short-code from name if needed
+    for item in message:
+        if item.length > 3:
+            crypto = [get_key(item)]
```

Comment:
* Are you expecting users to type the full name exactly? I'd suggest the arguments to the command should be the abbreviation, e.g. "!crypto btc" rather than "!crypto bitcoin".
* This assumes that all coin names are a single word (per line 22), but many coins have 2+ words in the name so would never be able to be looked up.
* Again, did you mean to make this a list? It causes the request on line 41 to fail if `item` is a list. Was this tested?